### DISCUSSION TITLE
Obtain information on clicking on or hovering over a pointcloud or mesh

### DISCUSSION THREAD

**Discussion by ac5113** [isl-org/Open3D#4627] [Q&A]
Is there any way to obtain the vertex coordinates(s) or vertex ids that are clicked on/hovered in a generated pointcloud or mesh?

**Comment by zhuyinheng** [comment]
checkout this demo. its basically unproject clicked pixel to world point in callback function.
https://github.com/isl-org/Open3D/blob/c096cb21961b5b8f062c5ec4dfbb419d20b37a81/examples/python/visualization/mouse_and_point_coord.py#L86
